provisionerd/provisionerd.go

```diff
@@ -213,7 +213,7 @@ connectLoop:
 			p.opts.Logger.Warn(p.closeContext, "coderd client failed to dial", slog.Error(err))
 			continue
 		}
-		p.opts.Logger.Info(p.closeContext, "successfully connected to coderd")
+		p.opts.Logger.Debug(p.closeContext, "successfully connected to coderd")
```

Comment:
It'd still be nice to log this at info in external provisioners since I think it's pretty useful, but you'll need to somehow differentiate whether we're using an in-memory dialer or a real dialer

Reply:
Doesn't really seem possible, unfortunately. I think this connected message is a lot more valuable than the starting provisioner message, so I'll swap them